ocrd_tesserocr/binarize.py

```diff
@@ -32,6 +32,13 @@ def __init__(self, *args, **kwargs):
         kwargs['ocrd_tool'] = OCRD_TOOL['tools'][TOOL]
         kwargs['version'] = OCRD_TOOL['version']
         super(TesserocrBinarize, self).__init__(*args, **kwargs)
+        if hasattr(self, 'output_file_grp'):
+            try:
+                self.page_grp, self.image_grp = self.output_file_grp.split(',')
```

Comment:
`split` results in a list, right? Not sure about the assignment here. What if we have only one output file group or three?

Reply:
Ah, both cases may result in `ValueError`. But the exception does not cover `A,B,C`.

Reply:
What do you mean the exception does not cover `A,B,C`? If anything except two output file groups are given, we enter the catch clause, as it should be. I don't see the problem...

Reply:
And when you enter the catch clause in the `A,B,C` case, you assign `self.page_grp` with `A,B,C`. How is that supposed to work?

Reply:
Oh, I see. Well, it is not supposed to work any better than without having all the candy here. That is, at some point later on, this will fail.

But I agree with you all these errors should be reported properly and as early as possible. So either the `Processor` base class or `process_cli` / `ocrd_cli_wrap_processor` should check in `ocrd-tool` for the number of outputs. We [discussed this in the spec](https://github.com/OCR-D/spec/issues/117), but it is not official, and not implemented yet. Also related to https://github.com/OCR-D/core/pull/296.

Reply:
It could be cleaner and will be, these issues will be adressed but for now, I don't think this is a showstopper.